### PR TITLE
fixes #368 - add support for scap_deployment_option_tab

### LINKED ISSUE #368: Add view for deployment_option tab of scap policy.
Tests related to scap policy creation are failing because there is new tab added for deployment option.

### PR DESCRIPTION
**Pull request by jameerpathan111**
Temporary fix for issue #368
This PR contains temporary fix for failing UI test foreman.ui.test_oscappolicy.test_positive_check_dashboard and some tier1 tests failing due to new changes in policy creation page.
Bugzilla [1731987](https://bugzilla.redhat.com/show_bug.cgi?id=1731987) has been filed to see whether it is possible to make compliance policy creation page consistent with rest of Satellite UI.

### CODE REVIEW

File: airgun/views/oscappolicy.py [hunk 1]
```diff
@@ -64,8 +65,18 @@ def is_displayed(self):
         )
 
     @View.nested
-    class create_policy(BaseLoggedInView):
-        TAB_NAME = 'Create policy'
+    class deployment_options(BaseLoggedInView):
+        TAB_NAME = 'Deployment Options'
+        next_step = Text("//input[contains(@value, 'Next')]")
+        deploy_by_ = RadioGroup(
```
Comment (omkarkhatavkar) [inline airgun/views/oscappolicy.py]: deploy_by_ the extra underscore was it intentional

Comment (jameerpathan111) [reply airgun/views/oscappolicy.py]: nope :)

**Comment by omkarkhatavkar** [comment]
@jameerpathan111 Please add test result

**Comment by jameerpathan111** [comment]
I am waiting for developer to comment on bugzilla [1731987](https://bugzilla.redhat.com/show_bug.cgi?id=1731987) and then I will update PR accordingly. Currently RadioGroup doesn't work(out of the box) for radio buttons of "deployment option" tab of compliance policy creation page.

File: airgun/views/oscappolicy.py (hunk 1 above)
Comment (vijay8451) [reply airgun/views/oscappolicy.py]: @jameerpathan111 can you keep this as `deploy_by` only ?

Comment (jameerpathan111) [reply airgun/views/oscappolicy.py]: yes

**Comment by jameerpathan111** [comment]
@omkarkhatavkar @vijay8451 don't we have WIP, do-not-merge labels for airgun?
Shall I create them?

**Comment by jameerpathan111** [comment]
> @omkarkhatavkar @vijay8451 don't we have WIP, do-not-merge labels for airgun?
> Shall I create them?

created issue #374

**Comment by mshriver** [comment]
@jameerpathan111 this looks abandoned to some degree, though there was plenty of discussion about an enhancement that would help our automation here and improve the page UX.  I'm going to move it to draft state for now, please monitor BZ https://bugzilla.redhat.com/show_bug.cgi?id=1731987 and update the PR as necessary should the PR get closed or accepted and re-triaged.

**Pull request closed without merging**